pom.xml

```diff
@@ -968,11 +968,16 @@
       <groupId>javax.xml.bind</groupId>
       <artifactId>jaxb-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
```

Comment:
I don't think this should be a dependency for every module, its used mostly for web modules.